TESTING.md

```diff
@@ -1,15 +1,15 @@
 - [Overview](#overview)
 - [Testing Search Functionality](#testing-search-functionality)
-  - [Test 1: Interactive Search Input](#test-1-interactive-search-input)
-  - [Test 2: Submitting a Search Query](#test-2-submitting-a-search-query)
+  - [Test 1](#test-1-interactive-search-input)
+  - [Test 2](#test-2-submitting-a-search-query)
 - [Testing Debug Panel Interactivity](#testing-debug-panel-interactivity)
-  - [Test 1: Default Data Source Selection](#test-1-default-data-source-selection)
-  - [Test 2: Interaction with Data Source Options](#test-2-interaction-with-data-source-options)
-  - [Test 3: Data Source Selection Affects Search Results](#test-3-data-source-selection-affects-search-results)
+  - [Test 1](#test-1-default-data-source-selection)
```

Comment:
don't agree with removing the text label description from the bullet point. What linting rule was it breaking?

Why not: 

```
[Test 1](#test-1-default-data-source-selection): Default Data Source Selection
```

Reply:
The markdown linting rule

Reply:
this would work:

```
[Test 1: Default Data Source Selection](
#test-1-default-data-source-selection)
```